ToDoList/ToDoList.API.Tests/Controllers/ToDosControllerTests.cs

```diff
@@ -0,0 +1,113 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using System.Web.Http;
+
+using NUnit.Framework;
+
+using ToDoList.API.Controllers;
+using ToDoList.API.Models;
+using ToDoList.API.Tests.Utilities;
+
+namespace ToDoList.API.Tests.Controllers
+{
+    [TestFixture]
+    public class ToDosControllerTests
+    {
+        private ToDosController _controller;
+        private List<ToDoModel> _toDoList;
+
+        [SetUp]
+        public void SetUp()
+        {
+            var httpConfiguration = new HttpConfiguration();
+            httpConfiguration.Routes.MapHttpRoute("GetToDo", "api/v1/todos/{id}", new {id = RouteParameter.Optional});
+
+            var httpRequestMessage = new HttpRequestMessage {RequestUri = new Uri("http://localhost:51200/api/v1/todos")};
+
+            _controller = new ToDosController
+            {
+                Request = httpRequestMessage,
+                Configuration = httpConfiguration
+            };
+
+            _toDoList = new List<ToDoModel>
```

Comment:
Do you really need this list to be reinitialized before every test? I understand that any changes to the list will thus be mitigated, however, making it a static array hidden behind an enumerable would prevent collection-wide changes (heavy code during review) and model changes should also be easy to spot in review…

I'm thinking about something like this:
````
private static readonly IEnumerable<ToDoModel> _toDoList = new []
{
   new …
};
````
If it was desired to always get items anew, I would expect a method that would yield them expressing the instability:
```
private static IEnumerable<ToDoModel> GetToDoList() 
{
    yield return new …
}
```